microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -3,15 +3,22 @@ description: Versioning Models used for Microsoft Edge WebView2
 title: Versioning of Microsoft Edge WebView2
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 10/19/2020
+ms.date: 10/28/2020
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
 keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
 ---
 # Understand WebView2 SDK versions  
 
-To develop a WebView2 application, you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload].  The minimum version that's required is included in the NuGet package version of the SDK.  For example, if you use the `SDK package version 0.9.488`, then you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload] with a build number of 488 or later.  The minimum version required is also specified in the WebView2 [Release Notes][Releasenotes].  New versions of the WebView2 SDK are shipped at the same general cadence as the Microsoft Edge \(Chromium\) browser, which is approximately every six weeks.  
+To develop a WebView2 application, you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload].  The minimum version required is specified in the WebView2 [Release Notes][Releasenotes]. For example, for SDK version [1.0.622.22](https://docs.microsoft.com/microsoft-edge/webview2/releasenotes#1062222), you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload] with a build number of **86.0.616.0** or later.
+
+However, new interfaces and APIs are added in later SDK versions. If your app depends on an API added in a later version, then your app may depend on a version of the runtime greater than the SDK's minimum version.
+
+You can find the minimum version required by an interface of API [programmatically](#matching-webview2-runtime-versions) or by looking up the minimum required version in the reference documentation.
```

Comment:
> by looking up the minimum required version in the reference documentation.

Does our API ref actually tell the min version of each API? I don't seem to find that information.

Reply:
you can for .NET today because it auto-generated from the NuGet package, we haven't built this into Win32

Reply:
Might want to call out we don't have that info for Win32 then to avoid win32 developer confusion. Where do I find this info for .NET?

Reply:
at the bottom of the page for every API

Reply:
It only tells you which SDKs contain the API. If I'm not familiar with WebView2, I probably wouldn't be able to infer the `minimum required version` from that.